src/routes/api.js

```diff
@@ -197,6 +198,9 @@ router.get('/devices', async (req, res) => {
                 </div>`;
                 device.uuid = `<a href='/device/manage/${device.uuid}' target='_blank' class='text-light'>${device.uuid}</a>`;
                 device.enabled = device.enabled ? 'Yes' : 'No';
+                const date = new Date();
+                const today = date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate();
+                device.game_restarts_today = await Stats.getAll(device.uuid + '-' + today + '-gamerestarts');
```

Comment:
```suggestion
                device.enabled = device.enabled ? 'Yes' : 'No';
                const date = new Date();
                const today = date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate();
                device.game_restarts_today = await Stats.getAll(device.uuid + '-' + today + '-gamerestarts');
                device.uuid = `<a href='/device/manage/${device.uuid}' target='_blank' class='text-light'>${device.uuid}</a>`;
```
`device.uuid` needs to be moved under game_restart_today otherwise it'll include the screenshot html text as the key, i.e: `<a href='/device/manage/iPhoneWM1SE' target='_blank' class='text-light'>iPhoneWM1SE</a>-2020-6-5-gamerestarts`

Seems to be working nicely after that change.

Reply:
Definitely, I forgot test when added column to devices panel :man_facepalming: 

Ty for fix that :partying_face: